### PR TITLE
gh-120780: Show attribute name for LOAD_SPECIAL in dis output

### PR DESCRIPTION
**Pull request by JelleZijlstra**
* Issue: gh-120780

### CODE REVIEW

File: Modules/_opcode.c [hunk 1]
```diff
@@ -349,6 +350,32 @@ _opcode_get_intrinsic2_descs_impl(PyObject *module)
 
 /*[clinic input]
 
+_opcode.get_special_methods
```
Comment (iritkatriel) [inline Modules/_opcode.c]:
```suggestion
_opcode.get_special_method_names
```

Comment (JelleZijlstra) [reply Modules/_opcode.c]: Thanks, done

File: Lib/dis.py [hunk 2]
```diff
@@ -609,6 +610,8 @@ def get_argval_argrepr(self, op, arg, offset):
                     argrepr = obj.__name__
                 else:
                     argrepr = repr(obj)
+            elif deop == LOAD_SPECIAL:
+                argrepr = get_special_method_names()[arg]
```
Comment (iritkatriel) [inline Lib/dis.py]: This creates the list every time. With the intrinsic descs we cache it in opcode.py.

Comment (JelleZijlstra) [reply Lib/dis.py]: Good point, fixed

**Review by iritkatriel — APPROVED** [review]

**Pull request merged**